src/vm/gcinfodecoder.cpp

```diff
@@ -648,7 +651,16 @@ bool GcInfoDecoder::EnumerateLiveSlots(
         //
         if(!executionAborted)
         {
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+            if(m_NumInterruptibleRanges == 0)
+            {
+                // No ranges and no explicit safepoint - must be MinOpts.
+                noTrackedRefs = true;
+                goto NO_RANGES;
+            }
```

Comment:
That's trivial to do (and I've tried it both ways) but when I asked earlier no feedback resulted so I left it alone; will do it by adding an 'if', then.